packages/google_maps_flutter/google_maps_flutter/example/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1320"
+   LastUpgradeVersion = "1300"
```

Comment:
@jmagman Did you want this reverted too? I'm using the latest stable Xcode so  if that number is related to that I'm not sure we should change it, but it's up to you.

Reply:
It doesn't really matter.  It should probably match what's in the Xcode project but Xcode is going to update it anyway when the next person touches it.  Whatever you prefer.